frontend/VolunteerOne/screens/Friends/notificationsPage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StyleSheet, Dimensions, ScrollView } from 'react-native';
+import { Block, theme } from 'galio-framework';
+import { Button } from "../../components";
+
+// import { Card } from '../../components';
+import NotificationItem from '../../components/NotificationItem';
+// import articles from '../../constants/articles';
+import notifications from '../../constants/notifications';
+
+// <matt>
+// import { Block, Button, Text, theme } from "galio-framework";
+// </matt>
+
+const { width } = Dimensions.get('screen');
+
```

Comment:
Please add a comment here that shows the name of the page, such as:
`/** ==================================== Notifications Page ==================================== **/
`

Reply:
committed